news/95.bugfix

```diff
@@ -0,0 +1 @@
+Disable automatic conversion of date strings in `from_dotlist`
```

Comment:
Can you change it to be not about dotlist?
This change is effecting all of OmrgaConf YAML decoding.

Reply:
I only checked `config.py` and thought that `merge_with_dotlist` is the only part that uses yaml loader, but you are indeed right! I'll fix that.